<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#Form{
	background-color:#EDF5ED;
}

QLabel{
	color: black;
}

QGroupBox{
	border: none;
	color: black;
}

QLineEdit, QDateEdit {
	border: 1px solid #91c3ad;
	border-radius: 7px;
	background-color: #deeee1;
	color: black;
}

QPushButton{
	background-color:#486161;
	border-radius: 10px;
	color: #f8f8f8;
	font-weight: bold;
}

QPushButton:hover{
	background-color: #5d8282;
	border: 2px solid #f8f8f8;
	color: #f8f8f8;
	font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>230</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="save_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Guardar</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../assets/icons/save_48dp.svg</normaloff>../assets/icons/save_48dp.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancel_btn">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Cancelar</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../assets/icons/cancel_48dp.svg</normaloff>../assets/icons/cancel_48dp.svg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>16</width>
        <height>16</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>58</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="id_txt">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="icon_label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../assets/icons/edit_document_48dp_black.svg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>261</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Fecha :</string>
      </property>
      <widget class="QDateEdit" name="date_process_dte">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Contrase√±a :</string>
      </property>
      <widget class="QLineEdit" name="pass_txt">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>241</width>
         <height>25</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
